samples/boards/google_pda/src/shell.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/usb/usb_device.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/drivers/usb/usb_dc.h>
+
+#include "view.h"
+#include "meas.h"
+#include "model.h"
+#include "mask.h"
+
+static int cmd_meas_cc2(const struct shell *shell, size_t argc, char **argv) {
+	int32_t out;
+
+	if (argc == 2 && *argv[1] == 'c') {
+		meas_vcon_c(&out);
+		shell_print(shell, "current of cc2: %i", out);
+	}
+	else {
+		meas_cc2_v(&out);
+		shell_print(shell, "voltage of cc2: %i", out);
+	}
+
+	return 0;
+}
+
+static int cmd_meas_vb(const struct shell *shell, size_t argc, char **argv) {
+	int32_t out;
+
+	if (argc == 2 && *argv[1] == 'c') {
+		meas_vbus_c(&out);
+		shell_print(shell, "current of vbus: %i", out);
+	}
+	else {
```

Comment:
Take a look at the style guide ... `else` must be cuddled. `clang-format` can fix this automatically for you.